packages/react-jss/.flowconfig

```diff
@@ -0,0 +1,11 @@
+[ignore]
```

Comment:
This file shouldn't exist because there is a flow config in the root for all packages